### PR TITLE
Implemented sendTransaction and sendRawTransaction benchmarking methods

### PR DESCRIPTION
**Pull request by illuque-iov**
Applied also some generic simplifications

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java [hunk 1]
```diff
@@ -20,26 +20,29 @@
...
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
 @Timeout(time = 20)
 public class BenchmarkWeb3 {
 
-    private static final Logger logger = LoggerFactory.getLogger(BenchmarkWeb3.class);
+    private static final int TRANSACTION_BATCH_SIZE = 16; // transaction.accountSlots = 16
 
     @State(Scope.Benchmark)
-    public static class ExecutionPlan {
+    public static class BasePlan {
```
Comment (rmoreliovlabs) [inline rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java:45]: It doesn't need to be now but feels like we shoukld put this on its own class

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java [hunk 2]
```diff
@@ -71,36 +73,99 @@ public void setUp() throws BenchmarkWeb3Exception {
...
+            transactionForEstimation = TransactionFactory.buildTransactionEstimation(properties, BigInteger.ONE);
         }
 
     }
 
-    @Benchmark
-    public void ethGetBalance(ExecutionPlan plan) throws BenchmarkWeb3Exception {
-        String address = (String) plan.properties.get("address");
+    @State(Scope.Benchmark)
+    public static class TransactionPlan extends BasePlan {
```
Comment (rmoreliovlabs) [inline rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java:83]: Same as above here

**Review by rmoreliovlabs — APPROVED** [review]
LGTM. Only 2 small nits that can be solved later

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java (hunk 1 above)
Comment (illuque-iov) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java:45]: Yes, valid point. Will do that in future changes on the feature branch.

**Pull request merged**